src/data-access/repositories/mongo/mongo-rundown-changed-listener.ts

```diff
@@ -25,7 +31,10 @@ export class MongoRundownChangedListener extends BaseMongoRepository implements
   constructor(
     mongoDatabase: MongoDatabase,
     mongoEntityConverter: MongoEntityConverter,
-    private readonly rundownRepository: RundownRepository
+    private readonly rundownRepository: RundownRepository,
+    private readonly rundownBaselineRepository: RundownBaselineRepository,
+    private readonly segmentRepository: SegmentRepository,
+    private readonly pieceRepository: PieceRepository
```

Comment:
I'm slightly concerned about injecting more repositories than the one be listening on, but I do see/understand why you are doing it.

It shouldn't become a habit though :)

Reply:
Noted

Reply:
I'm been made aware that some `NOT_FOUND` errors has been thrown from the database changed listeners on your branch.
I suspect very much that the changes in this file is the reason for it.